usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ValidateRouteColorAndTextContrastTest.java

```diff
@@ -85,7 +85,7 @@ void nullRouteTextColorShouldNotGenerateNotice() {
         verify(mockRoute, times(1)).getRouteColor();
         verify(mockRoute, times(1)).getRouteTextColor();
         verify(mockLogger, times(1)).info("Validating rule 'E025 - Insufficient route color" +
-                " contrast'"+ System.lineSeparator());
+                " contrast'" + System.lineSeparator());
```

Comment:
I think formatting character shall not end up in our JSON output

Reply:
I agree with @fabrice-v.

I think generally our JSON output is compact right now without any extra whitespace?

I've seen other tools implement a `--pretty` command-line parameter that adds new-lines at the end of each JSON line to make it easier to read. I think that would be a good option for this tool as well.

Reply:
Yes, right now we try to avoid any formatting into the output JSON.
Maybe we would implement prettyfiing but I would see it as low priority because there are tools, some of them online, that can also do it

Reply:
Agreed in terms of priority. Although, looks like enabling pretty print via Jackson is one line of code:
https://mkyong.com/java/how-to-enable-pretty-print-json-output-jackson/

The main effort on our end would be setting up and passing the command-line parameter.